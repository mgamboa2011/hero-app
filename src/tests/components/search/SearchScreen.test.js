// import { MemoryRouter, Routes } from 'react-router-dom';
// import { SearchScreen } from '../../../../src/components/search/SearchScreen';
// import { mount } from 'enzyme';

// describe('search', () => {
//     test('should ', () => {
//         const wrapper = mount(
//             <MemoryRouter children={<SearchScreen />} initialEntries={['/search']}>
//                 <Routes path='/search' element={<SearchScreen />} />
//             </MemoryRouter>
//         );
//         expect(wrapper).toMatchSnapshot();
//         expect(wrapper.find('.alert-info').text().trim()).toBe('');
//     })

// })
