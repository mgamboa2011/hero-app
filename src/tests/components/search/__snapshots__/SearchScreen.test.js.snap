// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`search should  1`] = `""`;
